js/script.js

```diff
@@ -0,0 +1,58 @@
+var searchData = [];
+
+$(document).ready(function(){
+
+var whatToSearch = "";
+var counter = 0;
+
+
+$('input').focus().val("");
+
+var appendPicture = function(){
+	if(searchData.data.children[counter].data.preview){
+			$('#slideshow').html("<img src='" + searchData.data.children[counter].data.preview.images[0].source.url + "' height='400px' width='400px'>");
+			counter++
+	}else{
+		clearInterval()
```

Comment:
calling clearInterval won't work here, because you aren't designating any interval to clear. Instead, I would just do something like
```
console.log('No preview image available.');
```